### PR TITLE
#444 - GetUserSlice introduced

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #444 
Added `Settings#credentials` method to return credentials yaml and introduced `GetUserSlice`.

### CODE REVIEW

File: src/main/java/com/artipie/api/artifactory/GetUserSlice.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            res =
+                new AsyncResponse(
+                    this.settings.credentials().thenApply(
+                        cred -> cred.map(
+                            yaml -> Optional.ofNullable(yaml.yamlMapping(username))
+                            .<Response>map(
+                                ignored ->
+                                    new RsWithBody(
+                                        new RsWithStatus(RsStatus.OK),
+                                        ByteBuffer.wrap(
```
Comment (g4s8) [inline src/main/java/com/artipie/api/artifactory/GetUserSlice.java:86]: @olenagerasimova you can use `new RsJson()` here from `artipie/http`

File: src/main/java/com/artipie/api/artifactory/GetUserSlice.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                                            Json.createObjectBuilder()
+                                                .add("name", username)
+                                                .add(
```
Comment (g4s8) [inline src/main/java/com/artipie/api/artifactory/GetUserSlice.java:89]: @olenagerasimova let's put required field such as `lastLoggedIn` and `realm` with stub values

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comments

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Comment by g4s8** [comment]
@olenagerasimova please resolve merge conflicts

**Comment by olenagerasimova** [comment]
@g4s8 conflicts resolved, please, merge

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#455` is not assigned, can't get performer

**Pull request merged**